### ISSUE TITLE
Dutch translation

### ISSUE THREAD

**Issue by zjean** [babybuddy/babybuddy-for-android#69]
Hi,
I would like to create a PR to add Dutch translations to the app. Is this doable? Any suggestions how implement this?

Regards

**Comment by MrApplejuice** [comment]
Hey zjean!

Yes, translations. I really would like to get them going, actually, but did not get around creating any because I gave priority to the "UI 2.0"-Design for now which will include tracking of ALL babybuddy event types. But if you have some spare time to provide some translations, I would be very happy to put a localized on the fast track and release it!

>  Any suggestions how implement this?

For Android the newbie-friendly way probably is to use the translation editor in Android Studio: https://developer.android.com/studio/write/translations-editor

I should have collected all relevant strings in one files at this point:

- https://github.com/babybuddy/babybuddy-for-android/blob/master/app/src/main/res/values/strings.xml
- Note: You might notice a few `translatable="false"` at the top however... translate those as well! Removing those magic strings is a left-over task for me!
- Note: Some strings contain patterns like: `Storing {activity} ...`. The `{activity}` part will be substituted by another localized string then.

The process should be relatively straight-forward and create a file called `res/values-nl/strings.xml` (for Dutch) which is the file I would need. With the file present, it should pretty much already be done for 90%. I did not pay very close attention to _how_ strings are formulated or exported at this point which is why I expect some teething problems with plural forms of words etc. but those can all be addressed relatively easily I think. Someone would just need to get this process started.
